Update typescript definitions

It looks like the typescript definitions are outdated in the latest version. I'm getting typescript errors such as 
```
error TS2339: Property 'setClientSecret' does not exist on type 'Dropbox'.
```
and
```
error TS2339: Property 'getAccessTokenFromCode' does not exist on type 'Dropbox'.
```

Can anyone point me in the right direction on how to update them? Thanks!

--- a/generator/typescript/dropbox_types.d.tstemplate
+++ b/generator/typescript/dropbox_types.d.tstemplate
@@ -21,7 +21,7 @@ declare module DropboxTypes {
      * @param state State that will be returned in the redirect URL to help
      *   prevent cross site scripting attacks.
      */
-    getAuthenticationUrl(redirectUri: string, state?: string): string;
+    getAuthenticationUrl(redirectUri: string, state?: string, authType = 'token'): string;
 
     /**
      * Get the client id

--- a/generator/typescript/dropbox_types.d.tstemplate
+++ b/generator/typescript/dropbox_types.d.tstemplate
@@ -39,6 +39,12 @@ declare module DropboxTypes {
      * @param clientId Your app's client ID.
      */
     setClientId(clientId: string): void;
+
+    /**
+     * Set the client secret
+     * @param clientSecret Your app's client secret.
+     */
+    setClientSecret(clientSecret: string): void;
   }
 
 /*TYPES*/

--- a/generator/typescript/dropbox_types.d.tstemplate
+++ b/generator/typescript/dropbox_types.d.tstemplate
@@ -15,6 +15,14 @@ declare module DropboxTypes {
     getAccessToken(): string;
 
     /**
+     * Get an OAuth2 access token from an OAuth2 Code.
+     * @param redirectUri A URL to redirect the user to after authenticating.
+     *   This must be added to your app through the admin interface.
+     * @param code An OAuth2 code.
+     */
+    getAccessTokenFromCode(redirectUri: string, code: string): Promise<string>;
+
+    /**
      * Get a URL that can be used to authenticate users for the Dropbox API.
      * @param redirectUri A URL to redirect the user to after authenticating.
      *   This must be added to your app through the admin interface.

--- a/src/dropbox-base.js
+++ b/src/dropbox-base.js
@@ -162,7 +162,7 @@ export class DropboxBase {
 
   /**
    * Set the client secret
-   * @arg {String} clientId - Your apps client secret
+   * @arg {String} clientSecret - Your app's client secret
    * @returns {undefined}
    */
   setClientSecret(clientSecret) {

